CONTRIBUTING.md

```diff
@@ -12,51 +12,64 @@ contributing enhancements to the Prometheus charm.
   implementation, you can reach us at
   [Canonical Mattermost public channel](https://chat.charmhub.io/charmhub/channels/charm-dev)
   or [Discourse](https://discourse.charmhub.io/).
-  The primary author of this charm is available on the Mattermost channel as
-  `@balbir-thomas`.
 - Familiarising yourself with the
   [Charmed Operator Framework](https://juju.is/docs/sdk)
   library will help you a lot when working on new features or bug fixes.
 - All enhancements require review before being merged. Code review
   typically examines
   + code quality
   + test coverage
-  + user experience for Juju administrators
-  this charm.
+  + user experience for Juju administrators.
```

Comment:
Do you think the phrase "Juju administrators" could be misunderstood?
If yes, how about:

```suggestion
  + cluster admin ux
```